content/hi/docs/reference/glossary/wg.md

```diff
@@ -0,0 +1,18 @@
+---
+title: WG (working group)
```

Comment:
- Consider moving “working group” to the `aka` section.
- Consider putting a translation of “working group” into the `aka` section as well.